src/utils.rs

```diff
@@ -42,6 +48,12 @@ impl AsRef<CurvePoint> for CurvePoint {
 }
 
 impl CurvePoint {
+    pub(crate) fn get_zero(&self) -> ProjectivePoint {
+        self.0
+    }
```

Comment:
note: Generally speaking, I don't like the name `get_zero` -- the "zero" this accesses is a function of the Rust tuple representation and doesn't tell us anything about what _type_ we're getting back or what the transformation does. I would rather implement `From<CurvePoint> for ProjectivePoint` or `AsRef<ProjectivePoint> for CurvePoint`.

However, I also think I've flagged all the places where you call this method to be turned into some other, more specific method, so I think this will just disappear.